graph_api/relationship/relationship_service.py

```diff
@@ -70,3 +92,38 @@ def save_relationship(self, relationship: RelationshipIn):
                                      name=relationship.name, errors={"errors": "not matching id"})
 
         return result
+
+    def save_properties(self, id: int, properties: List[PropertyIn]):
+        """
+        Send request to database by its API to create new properties
+
+        Args:
+            id (int): Id of the relationship
+
+            properties (List[PropertyIn]): List of properties for the relationship of given id
+
+        Returns:
+            Result of request as relationship object
+        """
+        if self.relationship_exist(id):
+            create_statement = f"MATCH ()-[r]->() where id(r)={id} set r = $props return r"
+            commit_body = {
+                "statements": [{"statement": create_statement,
+                                "parameters": {
+                                    "props": {
+                                        property.key: property.value for property in properties
+                                    }
+                                }}]
+            }
+            response = requests.post(url=self.database_url,
+                                     json=commit_body,
+                                     auth=self.database_auth).json()
+            if len(response["errors"]) > 0:
+                result = RelationshipOut(errors=response["errors"])
+            else:
+                relationship_id = response["results"][0]["data"][0]["meta"][0]["id"]
+                result = RelationshipOut(id=relationship_id, properties=properties)
+        else:
+            result = RelationshipOut(id=id, errors={"errors": "not matching id"})
+
+        return result
```

Comment:
According to PEP-8 there should be newline at the end of the file: https://pep8.org/